ResourcesCard.jsx

```diff
@@ -1 +1,83 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
```

Comment:
I am wondering if a `let` would be a better use for this `evt` since we are reassigning it. Or maybe we can just not create the object and reassign and instead move to what is in line 16.